src/ImageSharp/Formats/Jpeg/Components/GenericBlock8x8.Generated.cs

```diff
@@ -1,23 +0,0 @@
-﻿// Copyright (c) Six Labors.
```

Comment:
`GenericBlock8x8.Generated.tt` is an empty file -- should it be deleted?

Reply:
It should, looks like I forgot about it, thanks!

Reply:
This might be a github diff bug or something but I couldn't find `GenericBlock8x8.Generated.tt` file in my repo:

![image](https://user-images.githubusercontent.com/20967409/143986297-f6577e92-c49e-4de5-9972-a4b40c528350.png)

Reply:
Should be under `Format\Jpeg\Components`

Reply:
Yeah but screen above is `Format\Jpeg\Components`, here's another one directly from VS:
![image](https://user-images.githubusercontent.com/20967409/144007492-bfc74f41-c175-46ae-ae48-3b2e6d8913f4.png)

Sorry for my agnorance but what those `.Generated.tt` files mean exactly? First time encountering those.

Reply:
They're T4 templates generating partials of the named types. If you expand them you see the generated file. e.g. `Block8x8F.Generated.cs`. 

That empty one looks like a leftover from previous cleanup and is safe to delete.

Reply:
Completely missed your response - thanks for clarifying .tt stuff! `GenericBlock8x8.Generated.tt` seems to be deleted now (I did nothing after this review, looks like a diff tool bug or whatever).